pkg/apis/apis/v1alpha1/permissionclaims/labels.go

```diff
@@ -37,3 +37,50 @@ func ToLabelKeyAndValue(permissionClaim apisv1alpha1.PermissionClaim) (string, s
 	labelKeyHashLength := validation.LabelValueMaxLength - len(apisv1alpha1.APIExportPermissionClaimLabelPrefix)
 	return apisv1alpha1.APIExportPermissionClaimLabelPrefix + hash[0:labelKeyHashLength], hash, nil
 }
+
+func ValidateClaim(group, resource, identityHash string, grsToBoundResource map[apisv1alpha1.GroupResource]apisv1alpha1.BoundAPIResource) (bool, error) {
+	boundResource, bound := grsToBoundResource[apisv1alpha1.GroupResource{Group: group, Resource: resource}]
+	// This  says that if the resource is not bound, but there is no identity hash, we consider it valid
```

Comment:
This just restates the go code. But what does it mean? Why is not-bound and identity=="" value?

Reply:
updated